### PR TITLE
disable post editiong option enhanced globally detail #1163

### PR DESCRIPTION
**Pull request by zamandevs**

### CODE REVIEW

**Comment by FaraziF** [comment]
Please ensure the notice text when **pending** & **published** in both statuses is enabled. 

https://user-images.githubusercontent.com/7615279/146759165-82fbadaf-9ad4-4366-954e-29a7d26bab09.mov

File: admin/settings-options.php [hunk 1]
```diff
@@ -242,6 +242,13 @@ function wpuf_settings_fields() {
                     'type'    => 'checkbox',
                     'default' => 'on',
                 ],
+                [
+                    'name'    => 'disable_publish_edit',
+                    'label'   => __( 'Publish Post Edit', 'wp-user-frontend' ),
```
Comment (tareq1988) [inline admin/settings-options.php:247]: Change the label to `Editing Published Post`

**Comment by tareq1988** [comment]
@zamandevs Please resolve the conflict

**Comment by zamandevs** [comment]
> @zamandevs Please resolve the conflict

conflict resolved

**Pull request merged**